src/ca11y/index.js

```diff
@@ -95,6 +95,9 @@ class Ca11y {
    * @return { Void }
    **/
   setDate(date, silent=false) {
+
+    if(date.toString() === "Invalid Date") return
```

Comment:
Yeah, this felt brittle. If this check fails nothing bad actually happens other than an error logged in the console. Datepicker still opens to the last valid date or "today".